packages/database/src/migrations/20210610060144-AddFluTrackerLGAPercentNonFirstNationsILI-modifies-data.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+import { deleteObject, generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const INDICATOR = {
+  id: generateId(),
+  code: 'FluTracker_LGA_Percent_Non_First_Nations_ILI',
+  builder: 'analyticArithmetic',
+  config: {
+    formula:
+      'FWV_LGA_003b == -1 or FWV_LGA_003b == -1 or FWV_LGA_003b == 0 ? undefined : FWV_LGA_004b / FWV_LGA_003b',
+    aggregation: {
+      FWV_LGA_003b: [
+        'FINAL_EACH_DAY',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'sub_district',
+            aggregationEntityType: 'country',
+          },
+        },
+      ],
+      FWV_LGA_004b: [
+        'FINAL_EACH_DAY',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'sub_district',
+            aggregationEntityType: 'country',
+          },
+        },
+      ],
+    },
+    defaultValues: {
+      FWV_LGA_003b: -1,
+      FWV_LGA_004b: -1,
```

Comment:
I believe you could omit the `defaultValues` for this indicator? The normal indicator logic is to not produce an analytic unless all data elements are present for that period/entity. Since we only want the case where both data elements are present here, not using `defaultValues` is probably the way to go!

Reply:
Cool I will delete `defaultValues` here. So if not using `defaultValues` it will not produce analytic if all are presented?